lua/nvim-lsp-installer/_generated/metadata.lua

```diff
@@ -211,4 +211,4 @@ return {
   zls = {
     filetypes = { "zig", "zir" }
   }
-}
+}
```

Comment:
Btw @chen244 these files under the `_generated/` directory are automatically generated by running the `./scripts/autogen_metadata.sh` script :)